app/packs/src/components/structureEditor/StructureEditorContent.js

```diff
@@ -32,25 +32,24 @@ class StructureEditorContent extends React.Component {
     const { completionCallback, errorCallback } = this.props;
 
     perkinelmer.ChemdrawWebManager.attach({
-        id: 'chemdrawjs-container',
-        config: window.userConfiguration,
-        element: this.chemdrawcont,
-        callback: ()=> alert('done'),
-        errorCallback: ()=> alert('de'),
-        licenseUrl: 'license.xml',
-        // loadConfigFromUrl: 'cdjs/chemdrawweb/configuration.json',
-        // preservePageInfo: false,
-        // viewonly: true,
-      });
+      id: 'chemdrawjs-container',
+      config: window.userConfiguration,
+      element: this.chemdrawcont,
+      callback: () => alert('done'),
+      errorCallback: () => alert('de'),
+      licenseUrl: 'license.xml',
+      // loadConfigFromUrl: 'cdjs/chemdrawweb/configuration.json',
+      // preservePageInfo: false,
+      // viewonly: true,
```

Comment:
why are those commented?

Reply:
These things were in this file before.
And I can't really test it, because I can't load the editor.
It needs a license.